### PR TITLE
Unbiased traj

### PR DESCRIPTION
**Pull request by mehradans92**
- Fixed metapopulation model
- P0 study added to `NY_state_data_fitting.ipynb`
- P0 study was done for sparse graphs in `Patient_zero.ipynb` 
- Added new features to `utils.py`

### CODE REVIEW

File: maxentep/utils.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import scipy.stats as ss
 import matplotlib.pyplot as plt
 import maxentep
+import tensorflow as tf
```
Comment (whitead) [inline maxentep/utils.py:5]: Unused?

File: maxentep/maxent.py [hunk 2]
```diff
@@ -28,13 +28,13 @@ def traj_to_restraints(traj, inner_slice, npoints, prior, noise=0.1, time_averag
 
         def fxn(x, s=s, j=inner_slice):
             return tf.reduce_mean(x[s], axis=0)[j]
-        print(i * time_average + time_average // 2,
+        print(i * time_average + time_average // 2 + start_time,
```
Comment (whitead) [inline maxentep/maxent.py:31]: Probably not needed

File: maxentep/utils.py [hunk 3]
```diff
@@ -175,30 +179,187 @@ def merge_history(base, other, prefix=''):
     return base
 
 
-def compartment_restrainer(restrained_patches, restrained_compartments, npoints, ref_traj, prior, noise=0, start_time=None, end_time=None, time_average=7):
-    number_of_restrained_compartments = len(restrained_compartments)
+# def compartment_restrainer(restrained_patches, restrained_compartments, npoints, ref_traj, prior, noise=0, start_time=None, end_time=None, time_average=7):
```
Comment (whitead) [inline maxentep/utils.py:182]: Can we just remove this? We'll have the commented out source in commit history

**Review by whitead — CHANGES_REQUESTED** [review]

**Comment by whitead** [comment]
One this branch I cannot import the package due to this error:
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-c7d138e00ab2> in <module>
      1 import pandas as pd
----> 2 import maxentep
      3 from maxentep import MetaModel, traj_quantile, patch_quantile
      4 import numpy as np
      5 import tensorflow as tf

c:\users\white\repos\maxent-epidemiology\maxentep\__init__.py in <module>
      8 from .SIR_model import SIRModel
      9 from .metapop_model import *
---> 10 from .maxent import *

c:\users\white\repos\maxent-epidemiology\maxentep\maxent.py in <module>
      4 from math import sqrt
      5 from .utils import merge_history
----> 6 from keras import backend
      7 
      8 EPS = np.finfo(np.float32).tiny

ModuleNotFoundError: No module named 'keras'

```

File: maxentep/utils.py (hunk 1 above)
Comment (mehradans92) [reply maxentep/utils.py:5]: It is used in `compartment_restrainer` and  `get_dist` functions. I will add the `import` to the functions.

File: maxentep/maxent.py (hunk 2 above)
Comment (mehradans92) [reply maxentep/maxent.py:31]: I have made it so the user can define a time period for the observables to be selected from.

File: maxentep/utils.py (hunk 3 above)
Comment (mehradans92) [reply maxentep/utils.py:182]: removed

File: maxentep/utils.py (hunk 1 above)
Comment (whitead) [reply maxentep/utils.py:5]: It's fine, I just didn't notice the code that used it. Keep it in there.

**Review by whitead — APPROVED** [review]

**Pull request merged**